src/data/events.yml

```diff
@@ -986,3 +986,36 @@
     more_links:
         -   title: Všechny přednášky v&nbsp;playlistu
             url: https://www.youtube.com/watch?v=qE2QWIxWUv8&list=PLnXfazh66kVddq-0dEYgvD7JD-EIklEAF
+
+-   id: 30
+    slug: chrome-dev-tools-metriky-rychlosti-nacitani-jamstack
+    title: Chrome DevTools, metriky rychlosti načítání a&nbsp;JAMstack
+    description: >
+        Středa 14.&nbsp;3. 2018, Ataccama office Praha
+    speakers:
+        -   name: Iakov Markov
+            name_link: https://twitter.com/IakovMarkov
+            company: Ataccama
+            company_link: https://www.ataccama.com/
+            title: Chrome&nbsp;DevTools, základní i&nbsp;pokročilé měření výkonu
+            youtube_id: 8DCQAJWwhFE
+            image: iakov-markov.jpg
```

Comment:
… and here are Iakov's slides: https://docs.google.com/presentation/d/1b_V9H5vMOb2SkXh4Ev53hEADMlkM6FUL7HQriVtpbXI/edit#slide=id.p